javascript-light-bulb/main.js

```diff
@@ -0,0 +1,17 @@
+var $background = document.querySelector('.background');
+var $bulb = document.querySelector('.bulb');
+var currentClick = 0;
+
+$bulb.addEventListener('click', onAndOff);
+
+function onAndOff(event) {
+  currentClick++;
+
+  if (currentClick % 2 === 0) {
+    $background.className = 'background light';
+    $bulb.className = 'bulb on';
+  } else {
+    $background.className = 'background dark';
+    $bulb.className = 'bulb off';
+  }
```

Comment:
Instead of doing this with a counter, you should try doing this with a boolean. A good name for the variable would be `isOff` or `isOn`, with a boolean value matching the current state of the light bulb.